addons/editor-unshare-button/button.css

```diff
@@ -1,4 +1,4 @@
 .sa-unshare-button {
-  background: hsla(30, 100%, 55%, 1);
+  background: #ff4d4d;
```

Comment:
Please don't make unrelated changes in this pull request.

Reply:
Sorry! The reason why I made them on this branch is that the Minor text and design changes also were on the master branch. I'll undo it.